### ISSUE TITLE
New-lines in options cause formatting issues with title attribute (if multiple options selected).

### ISSUE THREAD

**Issue by PhilPJL** [snapappointments/bootstrap-select-temp#11]
New-lines in options cause formatting issues.

Windows 10
IE 11 / Edge
VS 2017
Bootstrap 3.3.7
Bootstrap-select 1.12.4

https://plnkr.co/edit/nBiCB4xZAP8Be4wzpj9A?p=preview

![image](https://user-images.githubusercontent.com/271656/36911640-06cc109c-1e3c-11e8-839e-6df815e02725.png)

You might say "don't put new lines in", but sometimes this happens during VS/Resharper auto formatting.

**Comment by caseyjhol** [comment]
I can't seem to recreate this in your linked example in Chrome, Edge, or IE11 on Windows 10. It almost looks like CSS or some JS isn't being loaded (as it doesn't make sense that none of the other options would be visible; even the "Close" button is missing).

**Comment by PhilPJL** [comment]
You need to select Apple and Banana.  This is on IE Edge although it's the same on any browser.
What you're seeing there is the tooltip.  There's also a space before the comma in the control itself.
It's only an issue with multiple selection.
![image](https://user-images.githubusercontent.com/271656/37049602-44a5011e-2169-11e8-9b1d-0d8e42773886.png)
What we should see is
![image](https://user-images.githubusercontent.com/271656/37049741-c20acb52-2169-11e8-8a6e-2ea77d828d87.png)

**Comment by caseyjhol** [comment]
Ah - thanks for the clarification. Should be an easy fix.

**Comment by PhilPJL** [comment]
Yes, I think something like this may do it (adding .trim()):

```
      if (typeof $this.attr('title') !== 'undefined') {
        return $this.attr('title');
      } else if ($this.data('content') && that.options.showContent) {
        return $this.data('content').toString();
      } else {
        return icon + $this.html()**.trim()** + subtext;
      }
    }
  }).toArray();
```

**Comment by caseyjhol** [comment]
Released in [v1.13.0](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.13.0)!
